halo/app/ready.go

```diff
@@ -0,0 +1,96 @@
+package app
+
+import (
+	"encoding/json"
+	"io"
+	"sync"
+
+	"github.com/omni-network/omni/lib/errors"
+)
+
+// Consist of all flags describing the current readiness status of a halo node.
+type readinessStatus struct {
+	// Indicates if the current halo instance uses an EVM RPC endpoint instead of running an execution client.
+	peersExposed bool
+	mu           sync.RWMutex
+	status       flags
```

Comment:
not sure about different name for field and type, can't it be the same?

Reply:
Yes, changed!